cli/types/cypress.d.ts

```diff
@@ -3113,6 +3112,12 @@ declare namespace Cypress {
      * @default null
      */
     experimentalSkipDomainInjection: string[] | null
```

Comment:
@cacieprins This `experimentalSkipDomainInjection` type should be removed.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/30770/commits/c03e62bc0835bae1e2e28ab94dd9b57837352ba2

Reply:
I think full removal of `experimentalSkipDomainInjection` should be in a follow-up PR, to keep the scope of this as manageable as possible